imap/itip_support.c

```diff
@@ -594,13 +594,13 @@ static int deliver_merge_request(const char *attendee,
     for (; itip; itip = icalcomponent_get_next_component(request, kind)) {
         icalcomponent *new_comp = icalcomponent_clone(itip);
 
-        /* Lookup this comp in the hash table */
+        /* Lookup and remove this comp from the hash table */
```

Comment:
Probably this should read "Lookup and remove this comp in the old object"?

Reply:
Actually we are just removing it from the hash table, NOT the old object (yet).  I added clarifying comment